forms-shared/src/definitions/formDefinitionTypes.ts

```diff
@@ -54,7 +55,12 @@ export type FormDefinitionSlovenskoSk =
 
 export type FormDefinitionEmail = FormDefinitionBase & {
   type: FormDefinitionType.Email
+  sendEmailFunction: 'sendOloEmail' | 'sendEmail'
   email: string
+  emailFrom?: string // If undefined, the `email` is used.
+  newSubmissionEmailTemplate: MailgunTemplateEnum
+  userEmailTemplate: MailgunTemplateEnum
+  sendJsonData?: boolean
```

Comment:
It's not clear from this naming if it will be attached to service/non-service email. I would also add "attachement" in the property name.

Reply:
I would also consider adding parametrizing whether to add "potvrdenie.pdf".